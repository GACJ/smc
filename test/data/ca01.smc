Cambridge
x38x14x1258x36x14x58x16x78 l12
5000 40320
0
