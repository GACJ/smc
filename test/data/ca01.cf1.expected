
5600 Cambridge Surprise Major, gen. SMC32 (No.1)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5152 Cambridge Surprise Major, gen. SMC32 (No.2)
  23456   B  M  W  H
  35264      2  2  3
  56342      2  2  3
  64523      2  2   
  42635      2  2  3
  23456      2  2  3
Score: 503
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  79 4 bell runs
  35 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  6 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5152 Cambridge Surprise Major, gen. SMC32 (No.3)
  23456   B  M  W  H
  35264      2  2  3
  56342      2  2   
  64523      2  2  3
  42635      2  2  3
  23456      2  2  3
Score: 491
  34 56s and 65s back and front
  89 crus back and front
  61 crus
  28 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  7 7658s front
  1 3468s
  18 3578s
  14 3478s
  79 4 bell runs
  37 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  18 5 bell runs
  7 5 bell runs front
  6 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (No.4)
  23456   B  M  W  H
  34256            2
  34562      2  2  -
5 part
Score: 489
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 2468s
  1 3468s
  4 3578s
  18 3478s
  76 4 bell runs
  38 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  9 5 bell runs front
  4 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)

5184 Cambridge Surprise Major, gen. SMC32 (No.5)
  23456   B  M  W  H
  35264      2  2  3
  56342   -        3
  56423      2  2  -
  25463         -   
  64352      -  -   
  64523   -        -
  42635      2  2  3
  23456      2  2  3
Score: 489
  34 56s and 65s back and front
  92 crus back and front
  60 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  73 4 bell runs
  35 4 bell runs front
  3 23456s
  10 3456s
  12 6543s
  12 2345s
  10 5432s
  19 5 bell runs
  6 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5184 Cambridge Surprise Major, gen. SMC32 (No.6)
  23456   B  M  W  H
  23564      2  2  -
  42563   -  -  -   
  35264      -     -
  56342      2  2  3
  64523   -        3
  42635      2  2  3
  23456      2  2  3
Score: 483
  34 56s and 65s back and front
  84 crus back and front
  52 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  13 3478s
  78 4 bell runs
  39 4 bell runs front
  2 23456s
  10 3456s
  12 6543s
  12 2345s
  12 5432s
  18 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5056 Cambridge Surprise Major, gen. SMC32 (No.7)
  23456   B  M  W  H
  35264      2  2  3
  56342   -        3
  64523      2  2  3
  42635      2  2  3
  23456      2  2  3
Score: 482
  33 56s and 65s back and front
  88 crus back and front
  58 crus
  30 crus front
  18 5678s
  8 5678s front
  3 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  70 4 bell runs
  39 4 bell runs front
  3 23456s
  10 3456s
  12 6543s
  12 2345s
  10 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5184 Cambridge Surprise Major, gen. SMC32 (No.8)
  23456   B  M  W  H
  35264      2  2  3
  56342      2  2  3
  56423      2  2  -
  35426   -  -  -   
  64523      -     -
  42635      2  2  3
  23456   -        3
Score: 482
  32 56s and 65s back and front
  92 crus back and front
  60 crus
  32 crus front
  16 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  16 3578s
  12 3478s
  75 4 bell runs
  34 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  10 5432s
  17 5 bell runs
  6 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5184 Cambridge Surprise Major, gen. SMC32 (No.9)
  23456   B  M  W  H
  35264      2  2  3
  56342      2  2  3
  64523   -        3
  64235      2  2  -
  36245         -   
  42563      -  -   
  42635   -        -
  23456      2  2  3
Score: 479
  34 56s and 65s back and front
  91 crus back and front
  61 crus
  30 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  2 7568s front
  7 7658s front
  1 3468s
  10 3578s
  13 3478s
  74 4 bell runs
  39 4 bell runs front
  2 23456s
  10 3456s
  12 6543s
  10 2345s
  12 5432s
  17 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5152 Cambridge Surprise Major, gen. SMC32 (No.10)
  23456   B  M  W  H
  35264      2  2  3
  56342      2  2  3
  64523      2  2  3
  42635      2  2   
  23456      2  2  3
Score: 476
  32 56s and 65s back and front
  89 crus back and front
  61 crus
  28 crus front
  18 5678s
  6 5678s front
  4 6578s
  4 6578s front
  2 7568s front
  7 7658s front
  1 3468s
  6 3578s
  13 3478s
  78 4 bell runs
  37 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

/ ************* Special selection ****************

5600 Cambridge Surprise Major, gen. SMC32 (Max 56s and 65s back and front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max crus back and front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max crus)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max crus front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 5678s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 5678s front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 6578s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 6578s front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 8765s)
  23456   B  M  W  H
  52436         -  3
  36524      2  -   
5 part
Score: 444
  26 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  14 5678s
  4 5678s front
  4 6578s
  4 6578s front
  2 8765s
  6 7568s front
  7 7658s front
  1 2468s
  14 3578s
  4 3478s
  64 4 bell runs
  34 4 bell runs front
  3 23456s
  6 3456s
  6 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  8 5 bell runs front
  5 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 7568s front)
  23456   B  M  W  H
  52436         -  3
  36524      2  -   
5 part
Score: 444
  26 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  14 5678s
  4 5678s front
  4 6578s
  4 6578s front
  2 8765s
  6 7568s front
  7 7658s front
  1 2468s
  14 3578s
  4 3478s
  64 4 bell runs
  34 4 bell runs front
  3 23456s
  6 3456s
  6 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  8 5 bell runs front
  5 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 7658s front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 2468s)
  23456   B  M  W  H
  34256            2
  34562      2  2  -
5 part
Score: 489
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 2468s
  1 3468s
  4 3578s
  18 3478s
  76 4 bell runs
  38 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  9 5 bell runs front
  4 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)

5600 Cambridge Surprise Major, gen. SMC32 (Max 3468s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 3578s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 3478s)
  23456   B  M  W  H
  34256            2
  34562      2  2  -
5 part
Score: 489
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 2468s
  1 3468s
  4 3578s
  18 3478s
  76 4 bell runs
  38 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  9 5 bell runs front
  4 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)

5600 Cambridge Surprise Major, gen. SMC32 (Max 4 bell runs)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 4 bell runs front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 23456s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 3456s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 6543s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 2345s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 5432s)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 5 bell runs)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 5 bell runs front)
  23456   B  M  W  H
  34256            2
  34562      2  2  -
5 part
Score: 489
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 2468s
  1 3468s
  4 3578s
  18 3478s
  76 4 bell runs
  38 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  16 5 bell runs
  9 5 bell runs front
  4 6 bell runs
  2 6 bell runs front
  1 7 bell runs (b&f)

5600 Cambridge Surprise Major, gen. SMC32 (Max 6 bell runs)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 6 bell runs front)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max 7 bell runs (b&f))
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums

5600 Cambridge Surprise Major, gen. SMC32 (Max Tittums)
  23456   B  M  W  H
  35264      2  2  3
5 part
Score: 510
  34 56s and 65s back and front
  94 crus back and front
  62 crus
  32 crus front
  18 5678s
  8 5678s front
  4 6578s
  4 6578s front
  3 7568s front
  9 7658s front
  1 3468s
  18 3578s
  14 3478s
  80 4 bell runs
  39 4 bell runs front
  3 23456s
  12 3456s
  12 6543s
  12 2345s
  12 5432s
  19 5 bell runs
  8 5 bell runs front
  6 6 bell runs
  3 6 bell runs front
  2 7 bell runs (b&f)
  1 Tittums
